presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -204,6 +204,7 @@
     public static final String MATERIALIZED_VIEW_DATA_CONSISTENCY_ENABLED = "materialized_view_data_consistency_enabled";
     public static final String QUERY_OPTIMIZATION_WITH_MATERIALIZED_VIEW_ENABLED = "query_optimization_with_materialized_view_enabled";
     public static final String AGGREGATION_IF_TO_FILTER_REWRITE_ENABLED = "aggregation_if_to_filter_rewrite_enabled";
+    public static final String DRIVER_EXPR_EVAL_SIMPLIFIED = "driver.expr_eval.simplified";
```

Comment:
I wonder if it makes sense to use PRESTISSIMO as the prefix for all C++-specific properties and preserve existing conventions such as using full words, e.g. PRESTISSIMO_SIMPLIFIED_EXPRESSION_EVALUATION_ENABLED = "simplified_expression_evaluation_enabled"

Reply:
I think it makes sense. I think as @highker mentioned we should eventually move prestissimo related session properties in cpp instead of java (I am not sure how to achieve that). Prefixing them makes later migration work easier

Reply:
> move prestissimo related session properties in cpp instead of java (I am not sure how to achieve that)

One option could be to build a mechanism in the coordinator to load extra session properties definitions from a file at startup. This would allow us to create a file with Prestissimo-specific properties and place it in the "right" location at deployment time.

Reply:
Or instead of a file, from cpp worker directly on startup. That way we don't need to maintain anything on coordinator on deployment time.